waffle-chai/src/types.ts

```diff
@@ -12,6 +12,8 @@ declare namespace Chai {
     properAddress: void;
     changeBalance(wallet: any, balance: any): AsyncAssertion;
     changeBalances(wallets: any[], balances: any[]): AsyncAssertion;
+    calledOnContract(contract: any): void;
+    calledOnContractWith(contract: any, ...parameters: any[]): void;
```

Comment:
This type is incorrect or the code is incorrect. The tests use it as `calledOnContractWith(contract, [1, 2]` so the type should be `calledOnContractWith(contract: any, parameters: any[])`